src/components/codeContext.tsx

```diff
@@ -176,6 +176,65 @@ export async function fetchCodeKeywords(): Promise<CodeKeywords> {
   };
 }
 
+function getCsrfToken(cookies = document.cookie): string | null {
+  // is sentry-sc in production, but may also be sc in other envs
+  // So we just try both variants
+  const cookieNames = ['sentry-sc', 'sc'];
+
+  return cookieNames
+    .map(cookieName => getCookie(cookies, cookieName))
+    .find(token => token !== null);
+}
+
+function getCookie(cookies: string, cookieName: string) {
+  const csrfCookie = cookies.split(';').find(cookie => {
+    const str = cookie.trim();
+    return str.startsWith(`${cookieName}=`);
+  });
+
+  return csrfCookie ? csrfCookie.split('=')[1] : null;
+}
```

Comment:
Let's no use this please. Let's use `js-cookie`. If we don't already have it we should add it